play-json/js/src/main/scala/StaticBinding.scala

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+package play.api.libs.json
+
+import java.io.InputStreamReader
+
+import scalajs.js, js.JSON
+
+object StaticBinding {
```

Comment:
StaticBinding, either JVM (Jackson for now) or JS (ScalaJS), for which the object `JSON` is a facade